### PR TITLE
Add upsell on library card and header gear. tracking

### PR DESCRIPTION
**Pull request by cvializ**
@camhashemi sanity check por favor. requested by @eishay

### CODE REVIEW

File: kifi-backend/modules/shoebox/angular/src/libraries/libraryHeader.tpl.html [hunk 1]
```diff
@@ -24,9 +24,41 @@
...
+              Only library owners can edit library settings.<br />Change this in <a class="kf-link" ui-sref="orgProfile.settings.team({ handle: library.org.handle })">settings</a>
+            </span>
+            <span ng-if="getMeOrg().viewer.membership.role === 'admin' && getMeOrg().config.settings.force_edit_libraries.setting === ORG_SETTING_VALUE.ADMIN">
+              Admins and the library owner can edit library settings.<br />Change this in <a class="kf-link" ui-sref="orgProfile.settings.team({ handle: library.org.handle })">settings</a>
+            </span>
+            <span ng-if="getMeOrg().viewer.membership.role === 'admin' && getMeOrg().config.settings.force_edit_libraries.setting === ORG_SETTING_VALUE.MEMBER">
+              All team members can edit library settings.<br />Change this in <a class="kf-link" ui-sref="orgProfile.settings.team({ handle: library.org.handle })">settings</a>
+            </span>
+          </span>
+          <span ng-if="getMeOrg().config.showUpsells">
```
Comment (camhashemi) [inline kifi-backend/modules/shoebox/angular/src/libraries/libraryHeader.tpl.html:52]: `getMeOrg()` can return `undefined` if the library is not in an org or if you are not in the org right?

Comment (cvializ) [inline kifi-backend/modules/shoebox/angular/src/libraries/libraryHeader.tpl.html:52]: that's right, but the `library.org.viewer.membership.role` reference in the `ng-if` above would prevent this from rendering in that case

**Comment by camhashemi** [comment]
:+1: seems like at some point it'd be a nice-to-have to put these paid plan upsells in a directive since they're pretty heavy and can be added per paid feature

**Comment by cvializ** [comment]
thanks :sunglasses: good idea, we do have `kf-upsell`, but this is a special case where we don't want to use the standard upsell icon. We could refactor the directive to take an arbitrary target with a `transclude`, but this is good enough for now.

**Pull request merged**